src/settings.js

```diff
@@ -22,6 +22,13 @@ class Settings extends SafeEventEmitter {
     const oldSettings = storage.get(SETTINGS_STORAGE_KEY);
     settings = {...defaultSettings, ...oldSettings};
 
+    if (oldSettings != null && oldSettings.version !== process.env.EXT_VER) {
```

Comment:
this should be below the following code block. additionally, it would be great if there was a way to combine this with upgrading the flags?